### ISSUE TITLE
Error: InvalidStateError: Failed to execute 'send' on 'RTCDataChannel': RTCDataChannel.readyState is not...

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#3200]
https://sentry.io/organizations/parabol/issues/1113439704/?referrer=github_plugin

```
Error: InvalidStateError: Failed to execute 'send' on 'RTCDataChannel': RTCDataChannel.readyState is not 'open'

InvalidStateError: Failed to execute 'send' on 'RTCDataChannel': RTCDataChannel.readyState is not 'open'
```

AC
- [ ] error is not fatal

effort: 11

**Comment by mattkrick** [comment]
oh hey, this is the one jordan ran into. we should keep this open

**Comment by jordanh** [comment]
@mattkrick says fixed in #3714
